sprity-aseprite/src/blend.rs

```diff
@@ -175,11 +175,7 @@ fn hsl_hue_baseline(back: Color, front: Color, opacity: u8) -> Color {
     let (r, g, b) = set_saturation(front.r_f64(), front.g_f64(), front.b_f64(), sat);
     let (r, g, b) = set_luminocity(r, g, b, lum);
 
-    normal(
-        back,
-        Color::from_slice_f64(&[r, g, b, front.a_f64()]),
-        opacity,
-    )
+    normal(back, Color::from(&[r, g, b, front.a_f64()]), opacity)
```

Comment:
You're not doing anything with the returned value?

Reply:
Scratch that. Reviewing Rust code in diff form is terrible. It's the last expression of the fn and a return. I know. :facepalm: